### DISCUSSION TITLE
Compare with command line

### DISCUSSION THREAD

**Discussion by guy-teube** [doublecmd/doublecmd#1663] [Suggestions]
Hi there,
I wonder if there is a way to compare and put the results in txt file (or other) in command line?
I saw in the documentation we can do 
```
/doublecmd -P L path/to/left/dir -P R /path/to/left/dir
```
But this opens the GUI. I would like to run the same command, with an output parameter like -o output.txt which contains the resultat in txt format (or other).
My need is to run this everyday, and send the result by e-mail (with an external script, not by doublecmd :) ) without to run the GUI tool.
Is it possible? I'va searched but find anything.
Thank you for your help.

**Comment by j2969719** [comment]
dont quite understand why gui ofm is needed for this. you can just use for example `ls -lA /some/cool/dir1 > /tmp/list1 && ls -lA /some/cool/dir2 > /tmp/list2 && diff -au /tmp/list1 /tmp/list2`
